playbooks/netgroup/netgroup-absent.yml

```diff
@@ -0,0 +1,11 @@
+---
+- name: Netgroup absent example
+  hosts: ipaserver
+  become: true
```

Comment:
Unless really required, I suggest using `become: no` and `gather_facts: no`.

`became: no` goes towards the "least privilege" rules, and `gather_facts: no` relates to performance.